### PR TITLE
add command line option for file system poll wait (seconds)

### PR DESCRIPTION
**Pull request by waichee**
Add an option on the command line for changing the default 30s file system poll wait interval.

### CODE REVIEW

**Comment by waichee** [comment]
I signed it!

**Comment by waichee** [comment]
Jenkins, test this please

**Comment by waichee** [comment]
@tensorflow-jenkins test this please

File: tensorflow_serving/model_servers/main.cc [hunk 1]
```diff
@@ -219,6 +221,11 @@ int main(int argc, char** argv) {
   core_config.aspired_version_policy =
       std::unique_ptr<AspiredVersionPolicy>(new EagerLoadPolicy);
 
+  // Override default file polling interval
+  if (file_system_polling_interval > 0) {
+    core_config.file_system_poll_wait_seconds = file_system_polling_interval;
```
Comment (chrisolston) [inline tensorflow_serving/model_servers/main.cc]: I'd prefer to give the flag the same name as the config variable, to minimize confusion. (Even if you may not like the name as much, I feel it's better to keep it consistent across the layers of the system.)

File: tensorflow_serving/model_servers/tensorflow_model_server_test.py [hunk 2]
```diff
@@ -68,13 +68,14 @@ def TerminateProcs(self):
     if self.server_proc is not None:
       self.server_proc.terminate()
 
-  def RunServer(self, port, model_name, model_path):
+  def RunServer(self, port, model_name, model_path, file_system_polling_interval):
```
Comment (waichee) [inline tensorflow_serving/model_servers/tensorflow_model_server_test.py]: Sure, ill leave the test case as only for the cases where the flag is omitted

**Comment by waichee** [comment]
@chrisolston  
- updated the command line option name to matching existing name
- revert test change

File: tensorflow_serving/model_servers/main.cc [hunk 3]
```diff
@@ -185,11 +185,13 @@ int main(int argc, char** argv) {
   tensorflow::int32 port = 8500;
   bool enable_batching = false;
   tensorflow::string model_name = "default";
+  tensorflow::int32 file_system_poll_wait_seconds = -1;
```
Comment (chrisolston) [inline tensorflow_serving/model_servers/main.cc]: I don't think -1 is (or should be) a valid value. Can we make the default 1 instead of -1 please.

**Comment by waichee** [comment]
sure updated from -1 to 1 and fix up description text

File: tensorflow_serving/model_servers/main.cc [hunk 4]
```diff
@@ -219,6 +221,11 @@ int main(int argc, char** argv) {
   core_config.aspired_version_policy =
       std::unique_ptr<AspiredVersionPolicy>(new EagerLoadPolicy);
 
+  // Override default file polling interval
+  if (file_system_poll_wait_seconds > 1) {
```
Comment (chrisolston) [inline tensorflow_serving/model_servers/main.cc]: Please drop the if-statement. (Setting it to 0 is valid, and the FileSystemStoragePathSource does (or at least should) do validation of the argument.)

Comment (waichee) [inline tensorflow_serving/model_servers/main.cc]: sure, if statement removed.

**Pull request merged**